cfme/common/vm.py

```diff
@@ -275,13 +275,19 @@ def open_console(self, console='VM Console', invokes_alert=None):
                            setting this to true will handle this.
         """
         # TODO: implement vmrc vm console
-        if console not in ['VM Console']:
+        if console not in ['VM Console', 'VMRC Console']:
             raise NotImplementedError('Not supported console type: {}'.format(console))
 
         view = navigate_to(self, 'Details')
 
-        # Click console button given by type
-        view.toolbar.access.item_select(console, handle_alert=invokes_alert)
+        # dismiss_any_alerts() call closed subsequent alerts needed for vmrc
+        # below code is needed to fix such issue
+        try:
+            view.browser.IGNORE_SUBSEQUENT_ALERTS = True
+            # Click console button given by type
+            view.toolbar.access.item_select(console, handle_alert=invokes_alert)
```

Comment:
Why the name change from `handle_alert` to `invokes_alert`? I would keep it consistent as `handle_alert`

Reply:
This is not being changed in the PR, its been like that always. And to me it makes sense because, when I am calling it like:
`vm.open_console(console='VM Console', invokes_alert=True)` 
it reads like, open VM console and that console invokes_alert. I think we can keep it as is unless you insist.

Reply:
That's fine with me, if you like it then we can keep it as is.